jekyll/_cci2/orb-development-kit.adoc

```diff
@@ -0,0 +1,31 @@
+---
+layout: classic-docs
+title: Orb development kit
```

Comment:
Could you add this to the sidebar in the Orb authoring section?

Reply:
Oh yeah, I thought I did that for some reason.